src/Mod/TechDraw/App/DrawRichAnno.cpp

```diff
@@ -65,24 +65,23 @@ DrawRichAnno::~DrawRichAnno()
 void DrawRichAnno::onChanged(const App::Property* prop)
 {
     if (!isRestoring()) {

```

Comment:
You're changing the logic here. Now it will return if `isRestoring()` is false, but earlier it would move forward.

Not sure how this improves readability either. As such just combining the two conditions with `&&` may be clearer.

Reply:
Thanks for attentive.